package.json

```diff
@@ -55,7 +55,8 @@
     "testcafe-reporter-xunit": "^1.0.0",
     "time-limit-promise": "^1.0.2",
     "useragent": "^2.1.7",
-    "uuid": "^2.0.1"
+    "uuid": "^2.0.1",
```

Comment:
I guess it makes sense to switch to short-id like we did in hammerhead: https://github.com/DevExpress/testcafe-hammerhead/blob/master/package.json#L31